ecl/hql/hqlgram.hpp

```diff
@@ -489,6 +488,8 @@ class HqlGram : implements IErrorReceiver, public CInterface
     IHqlExpression * createSortExpr(node_operator op, attribute & dsAttr, const attribute & orderAttr, HqlExprArray & args);
     IHqlExpression * createIffDataset(IHqlExpression * record, IHqlExpression * value);
     IHqlExpression * createSetRange(attribute & array, attribute & range);
+    bool checkDFSfields(IHqlExpression *dfsRecord, IHqlExpression *defaultRecord, const attribute& errpos);
```

Comment:
document: Should these new functions have desciptions of what they do?  (Several in this class)